content/sensu-core/2.0/guides/extract-metrics-with-checks.md

```diff
@@ -0,0 +1,133 @@
+product: "Sensu Core"
+platformContent: false
+menu:
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What are Sensu checks?
+
+In short, Sensu checks are **commands** (or scripts), executed by the Sensu
+agent, that output data and produce an exit code to indicate a state. If you are
+unfamiliar with checks, or would like to learn how to configure one first,
+take a look through the check [reference doc][1] and [guide][2] before you
+continue.
+
+## Extracting metrics from check output
+
+In order to extract metrics from check output, you'll need to do the following:
+
+1. Configure the check `command` such that the command execution will output
+metrics in one of the [supported metric formats][3].
+2. Configure the check `metric_format` to one of the [supported metric formats][3].
+3. Configure the check `metric_handler` (optional) to a Sensu handler that is
+equipped to handle Sensu metrics (see [handlers][4] or [influx-db handler][5]
+to learn more).
+
+You can configure the check with these fields at creation, or use the commands
+below assuming you have a check named `collect-metrics`. In this example,
+we'll be using `graphite_plaintext` format and sending the metrics to a handler
+named `influx-db`.
+
+{{< highlight shell >}}
+sensuctl check set-command collect-metrics collect_metrics.sh
+sensuctl check set-metric-format collect-metrics graphite_plaintext
+sensuctl check set-metric-handlers collect-metrics influx-db
+{{< /highlight >}}
+
+### Supported metric formats
+
+The metric formats that Sensu currently supports for check output metric
+extraction are nagios, influxdb, graphite, and opentsdb.
+
+nagios           |
+-----------------|------
+metric_format    | nagios_perfdata
+documentation    | [Nagios Performance Data][6]
+example          | `PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80`
+
+graphite         |
+-----------------|------
+metric_format    | graphite_plaintext
+documentation    | [Graphite Plaintext Protocol][7]
+example          | `local.random.diceroll 4 123456789`
+
+influxdb         |
+-----------------|------
+metric_format    | influxdb_line
+documentation    | [InfluxDB Line Protocol][8]
+example          | `weather,location=us-midwest temperature=82 1465839830100400200`
+
+opentsdb         |
+-----------------|------
+metric_format    | opentsdb_line
+documentation    | [OpenTSDB Data Specification][9]
+example          | `sys.cpu.user 1356998400 42.5 host=webserver01 cpu=0`
+
+### Validating the metrics
+
+If the check output is formatted correctly according to its `metric_format`,
+the metrics will be extracted in Sensu Metric Format, and saved within the
+event. You should expect to see logged errors if Sensu is unable to parse
+the check output. You can validate that metrics have been extracted from your
+check through your handler, or through the resulting event. The example check
+we used would yield an event similar to the one below:
+
+{{< highlight json >}}
+"event": {
```

Comment:
We need an opening bracket (`{`) here and a closing one at the end of this block!